components/main-chef-wrapper/cmd/env.go

```diff
@@ -33,14 +33,29 @@ var envCmd = &cobra.Command{
 	DisableFlagParsing: true,
 	RunE: func(cmd *cobra.Command, args []string) error {
 		if len(args) >= 1 {
-			cmd.Help()
-			os.Exit(0)
+			flags := []string{"--chef-license",
```

Comment:
is it possible --chef-license can passed after 1 argument it that case it will fails, should we handle that ?

Reply:
So chef env dosen't have any other sub-command, if --chef-license is passed after 1 argument  than in that case we will be displaying help message